### DISCUSSION TITLE
萨米肉鸽 还之彼身会关卡识别错误怎么办啊（还有一关不知道叫什么）

### DISCUSSION THREAD

**Discussion by sealant-fix** [MaaAssistantArknights/MaaAssistantArknights#7898] [Q&A]
萨米肉鸽 还之彼身会关卡识别错误怎么办啊（还有一关不知道叫什么）

**Comment by ABA2396** [comment]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new/choose
